src/lib.rs

```diff
@@ -39,7 +39,7 @@
 //! }
 //! ```
 //!
-//! Or alternatively, if you enable the `specs-derive` feature, you can use a
+//! Or alternatively, if you enable the `specs-derive` or `derive` feature, you can use a
```

Comment:
I think it's alright just mentioning the `"derive"` feature without mentioning `specs-derive`.